docs/source/developers/guide/step_by_step/testing.rst

```diff
@@ -29,3 +29,68 @@
 ***********
 Testing 🧪
 ***********
+
+In Arrow we always add a unit test to the code we are
+contributing. Therefore you will also need to add one when
+you finish with your work.
+
+.. tabs::
+
+   .. tab:: Pytest
+
+      We use `pytest <https://docs.pytest.org/en/latest/>`_ for
+      unit tests in Python. For more info about the required
+      packages follow
+      :ref:`Python unit testing section <python-unit-testing>`.
+
+      What we normally do is run the test we are working on
+      only. Once we are finished with our work and then 
+      we run other tests also.
```

Comment:
```suggestion
      During the development process, it can be more efficient to only run the tests relevant to the code which you are working on.  However, you should run all of the tests once you have finished your development work.
```
How about this?